cluster/ci/config/prow/config.yaml

```diff
@@ -127,9 +127,9 @@ presubmits:
     labels:
       master: ci.openshift.redhat.com
     always_run: true
-    context: ci/openshift-jenkins/extended_conformance_gce
```

Comment:
You should also update the SQ config to block on the new context. This will need everyone that has a green `ci/openshift-jenkins/extended_conformance_gce`  to rerun the job with the new context.

Reply:
We are getting rid of the SQ soon so the extra step should go away but renaming contexts is still an issue we haven't thought through.

Reply:
Following up with https://github.com/openshift/release/pull/556 and https://github.com/kubernetes/test-infra/issues/6368